### PR TITLE
Change GPT-2's Format to Mirror BERT's

### PR DESCRIPTION
**Pull request by abheesht17**
Resolves https://github.com/keras-team/keras-nlp/issues/404

### CODE REVIEW

File: keras_nlp/models/gpt2/gpt2_models.py [hunk 1]
```diff
@@ -26,17 +26,18 @@ def _gpt_2_kernel_initializer(stddev=0.02):
 
 
 @keras.utils.register_keras_serializable(package="keras_nlp")
-class Gpt2Custom(keras.Model):
-    """GPT-2 core network with customizable hyperparameters.
+class Gpt2(keras.Model):
+    """GPT-2 core network with custom hyperparameters.
```
Comment (jbischof) [inline keras_nlp/models/gpt2/gpt2_models.py:30]: We can remove the "custom" language here.

**Review by jbischof — APPROVED** [review]
Great work @abheesht17! Your attention to detail is impeccable!

**Comment by abheesht17** [comment]
> Great work @abheesht17! Your attention to detail is impeccable!

Thanks, @jbischof! :)

Removed "custom" from the doc-string.

**Review by mattdangerw — APPROVED** [review]
Thanks!

**Pull request merged**